django/apps/registry.py

```diff
@@ -10,6 +10,13 @@
 from .config import AppConfig
 
 
+class Tags(object):
```

Comment:
Ok, sorry for missing the point. Should the check (or SessionSettingChecks.check()?) be registered when the app is ready (like in contrib/admin/apps.py)?